includes/payment-methods/class-wc-stripe-upe-payment-method.php

```
@@ -543,9 +543,12 @@ public function can_refund_via_stripe() {
 	 *
 	 * @see WC_Stripe_UPE_Payment_Gateway::expand_copy_button_markup()
 	 *
+	 * @param bool $show_optimized_checkout_instruction Deprecated. Whether to show optimized checkout instructions.
+	 * @param bool $include_test_mode_label Whether to include the "Test mode:" label prefix. Pass false for
+	 *                                      Blocks checkout, which already displays a Test Mode badge.
 	 * @return string
 	 */
-	public function get_testing_instructions( bool $show_optimized_checkout_instruction = false ) {
+	public function get_testing_instructions( bool $show_optimized_checkout_instruction = false, bool $include_test_mode_label = true ) {
```

PHPMD (2.15.0)

[error] The method get_testing_instructions has a boolean flag argument $show_optimized_checkout_instruction, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)